### ISSUE TITLE
Using current masterfiles fail

### ISSUE THREAD

**Issue by varenius** [varenius/ascot#32]
eskil@grotte:~$ ivg_indep -v 5 -c ONTIE_groupdelay.cfg 
 
 =================================================================== 
     > > > > > ivg::ASCOT (independent solution) ver-2021-01-01 < < < < <     

terminate called after throwing an instance of 'std::runtime_error'
  what():  Masterfile::Masterfile(string directory, int start_year, int end_year): Unexpected field length of 16 with tokens IVS-R1875 R1875 JAN02  217:0024 in masterfile: /opt/ascot/apriori_files/masterfiles//master19.txtONN19JAN161.0  XA NASA  13 4639

This is with the official masterfile downloaded. Old files included in the ascot code bundle works. Now trying to figure out the difference...

**Comment by varenius** [comment]
This place in the masterfile.cpp is intersting:
```
                // erase first trash token
                tokens.erase(tokens.begin());
```
Printing the tokens to be parsed, before and after this statement, I find:
BEFORE:
```

IVS-R1875 
R1875 
JAN02
  2
17:00
24
HtIsKeKkKvNyOnWnYg -AgMa                                     
NASA
BONN
19JAN16
1.0 
 XA 
NASA
  13 
4639

```
AFTER:
```
IVS-R1875 
R1875 
JAN02
  2
17:00
24
HtIsKeKkKvNyOnWnYg -AgMa                                     
NASA
BONN
19JAN16
1.0 
 XA 
NASA
  13 
4639

```
The newline before the token list is gone. But the one at the end remains. I wonder if there has been a change of line endings or similar? In any case, adding 
```
                // erase last trash token
                tokens.erase(tokens.end());
```
works! Fixed in 5062f666c2852a7df318df2ced1d9a968f056a42.

**Comment by varenius** [comment]
Seems some old files are length 15, and new masterfiles length 16 tokens. Strange, well now actually fixed (although dirty) in 6f13c1993e361134096d10a7a6b27de0e652081c.
